FinniversKit/Sources/Components/MapAddressButton/MapAddressButton.swift

```diff
@@ -59,8 +59,10 @@ public class MapAddressButton: UIView {
         button.setImage(icon, for: .normal)
     }
 
-    public func setTitleColor(_ titleColor: UIColor) {
+    public func setButton(isEnabled: Bool) {
+        let titleColor: UIColor = isEnabled ? .textAction : .textDisabled
```

Comment:
I don't know why the properties on `Button.Style` is not public yet, but maybe we could make them public and rather here use `buttonStyle.titleColor` instead of `.textAction` so we don't unintentionally override the style we set above.

Reply:
Make sense
`Button.Style `is inside `public extension` so no need in more `public` 
Updated, does it looks better now?
p.s. Tested both with active & inactive = looks the same

Reply:
The struct `Button.Style` is public, but the properties are not – they are only internal.

Yes, now the colors will be in sync if we update the button style 🙂 👌

Reply:
So you basically saying it is smth new? :)
<img width="1364" alt="Screenshot 2023-12-19 at 09 40 28" src="https://github.com/finn-no/FinniversKit/assets/7100327/b835bcf1-8e3c-4f21-b4d5-d995fca60884">

Reply:
No, this has been like this for a long time. I'm talking about the properties, not the structs themselves.

<img width="739" alt="Screenshot 2023-12-19 at 09 44 01" src="https://github.com/finn-no/FinniversKit/assets/1901556/58d3e978-f54d-4d77-978d-4f3f87e4bb2b">

Reply:
But it's not that important now – you're able to access them since what you're adding is within FinniversKit.

Reply:
👌